content/docs/09-registries-and-packs/4.5-create-pack.md

```diff
@@ -0,0 +1,718 @@
+
+
+<br />
+
+![Visual of Cloud Accounts in the Palette console.](/tutorials/create-pack/registries-and-packs_create-pack_palette-cloud-account.png)
+
+<br /> <br />
+
+
+2. **SSH key**: An SSH key created in the region where you will deploy the cluster. This tutorial will deploy the cluster in the **us-east-2** region and assumes an SSH key, **aws_key_sk_us_east_2**, exists already in the AWS console. 
+
+<br /> <br />
+
+<Tabs>
+
+<Tabs.TabPane tab="Palette Console" key="Palette_Console">
+
+
+## Create a Profile
+Switch to the **Default** view and select the **Profile** section in the left **Main Menu** to create a full profile that combines core infrastructure and add-on layers. Click on the **Add Cluster Profile** button, and provide the details in the wizard that follows. The wizard will present you with several sections. 
+<br />
+
+### Section: Basic Information
+Use the following values in the first section, **Basic Information**. 
+
+|**Field**|**Value**|
+|---|---|
+|Name|*pack-tutorial-profile*|
+|Version|`1.0.0`|
+|Description|*Cluster profile as part of the pack tutorial.*|
+|Type|*Full*|
+|Tags|`spectro-cloud-education, app:hello-universe, terraform_managed:true`|
+
+<br />
+
+### Section: Cloud Type
+In the next section, **Cloud Type**, choose AWS as the infrastructure provider for this tutorial (for a consistent experience). If you choose any other cloud service provider, the core infrastructure layers options, as outlined in the table below, will differ from this tutorial. 
+
+<br />
+
+### Section: Profile Layers
+In the **Profile Layers** section, add the following core infrastructure layers:
+
+|**Pack Type**|**Registry**|**Pack Name**|**Pack Version**| 
+|---|---|---|---|
+|OS|Public Repo|Ubuntu|`LTS__20.4.x`|
+|Kubernetes|Public Repo|Kubernetes|`1.24.x`|
+|Network|Public Repo|Calico|`3.25.x`|
+|Storage|Public Repo|Amazon EBS CSI|`1.16.x`|
+
+Once you confirm adding the core infrastructure layers, click the **Add New Pack** button to add the following layers (including the custom add-on pack you defined and pushed to the registry). 
+
+|**Pack Type**|**Registry**|**Pack Name**|**Pack Version**| 
+|---|---|---|---|
+|Authentication | Public Repo| Spectro Proxy | `1.3.x`|
+| App Services | **private-pack-registry** | **Hello Universe** | `1.0.x` |
+
+Note that the values, **private-pack-registry** and the **Hello Universe**, may vary if you have chosen a different name for your registry and pack, respectively. 
+
+
+<br />
+
+![Visual of the Profile Layers success.](/tutorials/create-pack/registries-and-packs_create-pack_profile-layer.png)
+
+<br /> <br />
+
+
+### Section: Review
+Lastly, review and finish creating the profile configuration. You would see the newly created profile if there were no errors or compatibility issues between the individual layers. 
+
+<br /> <br />
+
+## Create a Cluster
```

Comment:
Same feedback as above. We are missing steps in the following section where you tell the user what to click. Assume the reader is a complete beginner brand new to Palette.